llvm/test/CodeGen/AMDGPU/llvm.amdgcn.wavefrontsize.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
The tests for this should go in test/InstCombine/AMDGPU

Reply:
Done.